judgels-backends/uriel/uriel-app/src/main/java/judgels/uriel/contest/ContestStore.java

```diff
@@ -86,13 +87,23 @@ public Contest createContest(ContestData contestData) {
         return fromModel(contestDao.insert(model));
     }
 
+    public ContestDescription getContestDescription(Contest contest) {
```

Comment:
- Instead of handling nulls like this, return an Optional instead, and then use checkFound() again in the resource
- We can just pass the contest jid as the method parameter